content/22.vaccines-trad.md

```diff
@@ -0,0 +1,410 @@
+Instead, 45.9% of cases were identified as the delta variant, 43.4% as gamma, 4.9% as alpha, and 5.8% as VOIs.
+Therefore, the efficacy data from this phase III trial may be lower than it would have been if the trial had occurred earlier in the course of SARS-CoV-2's evolution given that the S glycoprotein expressed in the VLPs was isolated from a 2020 sample of SARS-CoV-2 [@doi:10.1056/NEJMoa2201300].
+
+#### Other Concerns in Efficacy
+
+Given the wide range of vaccines under development, it is possible that some vaccine products may eventually be shown to be more effective in certain subpopulations, such as children, pregnant women, immunocompromised patients, the elderly, etc.
+
+Age distribution in clinical trials? https://doi.org/10.1016/j.arr.2021.101455
+
+Concerns: diversity of volunteer pools, variants, and distribution
+Another benefit of vaccines is lower population size in SARS-CoV-2 = less risk of VOC emerging that are less susceptible to the vaccine
+
+Given the apparent need for boosters, interest has also emerged in whether vaccines against SARS-CoV-2 can be administered along with annual flu vaccines.
+Early data came from the Novavax NVX-CoV2373 protein subunit vaccine.
+In a subgroup of approximately 400 patients enrolled from the U.K. phase III trial who received either NVX-CoV2373 or placebo 1:1, a concomitant dose of adjuvanted seasonal influenza vaccines (either a trivalent vaccine or a quadrivalent vaccine) was administered [@doi:10.1101/2021.06.09.21258556]. 
+This study demonstrated that both types of vaccines could be safely administered together. 
+While no change to the immune response was noted for the influenza vaccine, a notable reduction of the antibody response for the NVX-CoV2373 was reported, but efficacy was still high at 87.5% [@doi:10.1101/2021.06.09.21258556].
+Novavax has since started phase I/II trials to investigate the administration of its own influenza vaccine, NanoFlu^™^, concomitantly with NVX-CoV2373 [@url:https://ir.novavax.com/2021-09-08-Novavax-Initiates-Phase-1-2-Clinical-Trial-of-Combination-Vaccine-for-COVID-19-and-Seasonal-Influenza], which appeared to be safe and effective in preclinical studies [@doi:10.1101/2021.05.05.442782].
+<!--May end up moving some of this text to app, but worth discussing ongoing efforts?-->
+
+Indeed, Chinese [@url:https://medicalxpress.com/news/2021-08-china-mixed-vaccine-trial-delta.html] and Chilean [@url:https://clinicaltrials.gov/ct2/show/NCT04992182] researchers have opted to investigate options to administer different vaccines (e.g., an mRNA vaccine dose) as a booster dose to individuals who have already received two doses of the IWV vaccine CoronaVac.
+Another study has already determined that using the CanSino vaccine (Convidecia) instead of CoronaVac in a prime-boost vaccination regimen can induce a more robust immune response [@doi:10.1101/2021.09.03.21263062].<!--Add 10.1038/s41591-022-01705-6-->
+
```

Comment:
Concern about understanding of efficacy in the general public.... preventing disease vs infection... the negative connotations of breakthrough infections, when they are actually expected infections with limited disease/illness

Reply:
```suggestion
<!--Concern about understanding of efficacy in the general public.... preventing disease vs infection... the negative connotations of breakthrough infections, when they are actually expected infections with limited disease/illness-->
```